pkg/controller/datavolume/util.go

```diff
@@ -303,6 +305,73 @@ func resolveVolumeSize(c client.Client, dvSpec cdiv1.DataVolumeSpec, pvcSpec *v1
 	return &requestedSize, err
 }
 
+// storageClassCSIDriverExists returns true if the passed storage class has CSI drivers available
+func storageClassCSIDriverExists(client client.Client, log logr.Logger, storageClassName *string) (bool, error) {
+	log = log.WithName("getCsiDriverForStorageClass").V(3)
```

Comment:
wrong logged name

Reply:
I just moved the function but sure ill fix :)